testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -2950,4 +2950,34 @@ QUnit.module('itemRendered event', () => { // T906117
             assert.equal(expectedItemsArray.length, 0);
         });
     });
+
+    QUnit.test('itemRendered callback is called for menu & treeview items, adaptivityEnabled: true (T1092214)', function(assert) {
+        assert.expect(11);
+        const onItemRenderedHandler = sinon.stub();
+
+        $('#menu').dxMenu({
+            dataSource: testDataSource,
+            adaptivityEnabled: true,
+            width: 50,
+            onItemRendered: onItemRenderedHandler
+        }).dxMenu('instance');
+
+        const checkRenderedItem = (call, itemText, itemClass) => {
+            assert.strictEqual(onItemRenderedHandler.getCall(call).args[0].itemData.text, itemText);
```

Comment:
we can move `onItemRenderedHandler.getCall(call).args[0]` to a variable to reduce copypaste

Reply:
https://github.com/DevExpress/DevExtreme/pull/21782/commits/6c7393b5ca46b5cef91378fa4a6c4fc812168362